### PR TITLE
[4.2] New option in debug plugin in J4.2.4

### LINKED ISSUE #2653: [4.2] s/periond/period
New language relevant PR in upstream repo: https://github.com/joomla/joomla-cms/pull/39072 Here are the upstream changes: 
<details>
<summary>Click to expand the diff!</summary>

```diff
diff --git a/administrator/language/en-GB/plg_system_debug.ini b/administrator/language/en-GB/plg_system_debug.ini
index fe9220df4237..87e08c6a9853 100644
--- a/administrator/language/en-GB/plg_system_debug.ini
+++ b/administrator/language/en-GB/plg_system_debug.ini
@@ -32,7 +32,7 @@ PLG_DEBUG_FIELD_STRIP_PREFIX_LABEL="Strip From Start"
 PLG_DEBUG_FIELD_STRIP_SUFFIX_DESC="Strip words from the end of the string. For multiple words, use the format: (word1|word2)."
 PLG_DEBUG_FIELD_STRIP_SUFFIX_LABEL="Strip From End"
 PLG_DEBUG_FIELD_TRACK_REQUEST_HISTORY_LABEL="Track Request History"
-PLG_DEBUG_FIELD_TRACK_REQUEST_HISTORY_DESC="When enabled this saves the request history data into the filesystem for future analysis. This data may include passwords and secret tokens. Only enable this for a short periond of time as the file can be read by anyone with read access to the cache folder. When this setting is subsequently disabled it is strongly recommended to clear the Site Cache!"
+PLG_DEBUG_FIELD_TRACK_REQUEST_HISTORY_DESC="When enabled this saves the request history data into the filesystem for future analysis. This data may include passwords and secret tokens. Only enable this for a short period of time as the file can be read by anyone with read access to the cache folder. When this setting is subsequently disabled it is strongly recommended to clear the Site Cache!"
 PLG_DEBUG_LANG_LOADED="Loaded"
 PLG_DEBUG_LANG_NOT_LOADED="Not loaded"
 PLG_DEBUG_LANGUAGE_FIELDSET_LABEL="Language"

```
</details>

### PR DESCRIPTION
**Pull request by heelc29**
Pull Request für [Commit](https://github.com/joomla/joomla-cms/commit/8927b3d685bebae9046cd88239472a85a74d13f1) .

### Zusammenfassung der Änderungen

add strings

### Wo wird der Sprachstring angezeigt / Wie kann getestet werden

backend (com_plugins/plugin/edit)
image

### CODE REVIEW

**Review by tecpromotion — DISMISSED** [review]

**Comment by tecpromotion** [comment]
@joomlagerman/maintainers should we build a 4.2.4v2?
Otherwise it will be in the regular update in 14 days.

**Comment by zero-24** [comment]
I would say do it in the regular release cyclus. Its nothing super critical

File: administrator/language/de-DE/plg_system_debug.ini [hunk 1]
```diff
@@ -32,6 +32,8 @@ PLG_DEBUG_FIELD_STRIP_PREFIX_DESC="Wörter am Anfang des Strings entfernen. Für
 PLG_DEBUG_FIELD_STRIP_PREFIX_LABEL="Vom Anfang entfernen"
 PLG_DEBUG_FIELD_STRIP_SUFFIX_DESC="Wörter am Ende des Strings entfernen. Für Mehrwort-Strings folgendes Format anwenden: (Wort1|Wort2)"
 PLG_DEBUG_FIELD_STRIP_SUFFIX_LABEL="Am Ende entfernen"
+PLG_DEBUG_FIELD_TRACK_REQUEST_HISTORY_LABEL="Anfrageverlauf verfolgen"
+PLG_DEBUG_FIELD_TRACK_REQUEST_HISTORY_DESC="Wenn aktiviert, werden die Daten des Anfrageverlaufs für zukünftige Analysen im Dateisystem gespeichert. Diese Daten können Passwörter und geheime Token enthalten. Nur für kurze Zeit aktivieren, da jeder mit Lesezugriff auf das Cache-Verzeichnis die Datei lesen kann. Wenn diese Einstellung nachträglich deaktiviert wird, wird dringend empfohlen, den Seiten-Cache zu löschen!"
```
Comment (zero-24) [inline administrator/language/de-DE/plg_system_debug.ini]:
```suggestion
PLG_DEBUG_FIELD_TRACK_REQUEST_HISTORY_DESC="Wenn aktiviert, werden die Daten des Anfrageverlaufs für zukünftige Analysen im Dateisystem gespeichert. Diese Daten können Passwörter und geheime Token enthalten. Daher diese Option nur für kürzeste Zeit aktivieren, da jeder mit Lesezugriff auf das Cache-Verzeichnis die Dateien lesen können. Wenn diese Einstellung deaktiviert wird, wird außerdem dringend empfohlen, den Seiten-Cache zu löschen!"
```

Comment (tecpromotion) [inline administrator/language/de-DE/plg_system_debug.ini]:
```suggestion
PLG_DEBUG_FIELD_TRACK_REQUEST_HISTORY_DESC="Wenn aktiviert, werden die Daten des Anfrageverlaufs für zukünftige Analysen im Dateisystem gespeichert. Diese Daten können Passwörter und geheime Token enthalten. Diese Option nur für kurze Zeit aktivieren, da jeder mit Lesezugriff auf das Cache-Verzeichnis die Dateien lesen kann. Wenn diese Einstellung deaktiviert wird, wird außerdem dringend empfohlen, den Seiten-Cache zu löschen!"
```

**Comment by tecpromotion** [comment]
This PR fix also #2653

**Review by zero-24 — APPROVED** [review]

**Comment by zero-24** [comment]
Merging danke :+1:

**Pull request merged**